pkg/sysregistriesv2/system_registries_v2_darwin.go

```diff
@@ -0,0 +1,9 @@
+package sysregistriesv2
```

Comment:
Could we use build conditionals instead of file suffixes?

Two files should be sufficient, one with `// +build freebsd` the other one with `!freebsd`.